### ISSUE TITLE
False negative "UselessCallOnNotNull" with `list.isNullOrEmpty()`

### ISSUE THREAD

**Issue by t-kameyama** [detekt/detekt#3474]
```kotlin
fun test(list: List<Int>) {
    list.isNullOrEmpty()
}
```
